scripts/ci/create-binaries.sh

```diff
@@ -21,6 +21,7 @@ cd ${GOPATH}/src/github.com/skydive-project/skydive
 
 echo "--- BINARIES ---"
 make static WITH_EBPF=true
+make contribs
```

Comment:
I think `make contrib` is no longer needed because `make static` calls `contribs.static` (after calling `compile.static`).

Reply:
yup (removed)